mlibrary/my_account
# [LIBSEARCH-931] Update presentation of checkout history download and opt-out options

# Overview
The UX team has [updated the Settings page](https://www.figma.com/file/FOPCoTJX6PzEmbD6pjT8v2/Account?type=design&node-id=1105%3A2820&mode=dev&t=ReRMMgkDZYG8ulKg-1). `Text Notifcations` and `Checkout History` have been flipped. The text has also been updated.

This pull request closes [LIBSEARCH-931](https://mlit.atlassian.net/browse/LIBSEARCH-931).

## Testing
- Run the tests to make sure they pass (`docker-compose run --rm web bundle exec rspec`).
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- Checkout [Settings](http://localhost:4567/settings) and see if the new styles have been applied. Go through the rest of the site and see if anything is broken.
